playbooks/responses/stale-pr.md

```diff
@@ -0,0 +1,19 @@
+---
+action:
+ - add 'stale' label
+ - close pull request
+---
+
+Thank you for your contribution to the project!
+
+We apologize for the delay in reviewing your pull request. In the time since it
```

Comment:
> We apologize for the delay in reviewing your pull request

I the case of most stale PRs, it isn't usually that there is a delay in reviewing, but rather a lack of response from the author of the PR.

Reply:
Good point. I've updated the wording and made a few other minor adjustments.